nss-sapphire-elephants/BangazonAPI
# Orders

Developers can now access orders, query for completed orders, the customer on an order, and the product details on the order.

## Type of change



# Testing Instructions for Change Made

```git fetch --all```

```git checkout orders```

Test that you can:

``GET```
```POST```
```PUT```
```DELETE```

an order.

Test that you can see customers and products with the "?include=products" and the "?include=customers".

Test that user can see uncompleted orders with the "?completed=false" command.

# Checklist:
